routes/index.js

```diff
@@ -7,4 +7,15 @@ router.get('/', function (req, res, next) {
   res.render('index', { products: products });
 });
 
+router.get('/fiveisprime', function (req, res, next) {
```

Comment:
Looks like `req` and `next` are not needed here. Can we get rid of those somehow? Perhaps destructure or at least lose the `next` parameter?